helm/block-producer/templates/_healthchecks.tpl

```diff
@@ -65,7 +65,7 @@ readinessProbe:
     command: [
       "/bin/bash",
       "-c",
-      "source /healthcheck/utilities.sh && isDaemonSynced && peerCountGreaterThan 0"
+      "source /healthcheck/utilities.sh && isDaemonSynced && hasSentUserCommandsGreaterThan 0"
```

Comment:
are these changes accidental?

Reply:
Nope, intentional - I think this could be a product of desynced helm changes in coda-automation and mina? It makes sense in context too, its on the userAgent container

Reply:
Oh intentional and just a change of checks for user-agents. Since they're (usually) paired with a  daemon figured it made sense to leave the daemon checks to the `daemon containers`. Actually should probably remove the `isDaemonSynced` check also but I could see either way,